common/ethnicities/wc_ethnicities_goblin.txt

```diff
@@ -0,0 +1,471 @@
+	}
+	gene_bs_eye_size = {
+        1 = { name = eye_size_pos    range = { 0 1 }    }
+		1 = { name = eye_size_neg    range = { 0 1 }    }
+	}
+	gene_bs_eye_upper_lid_size = {
+        1 = { name = eye_upper_lid_size_pos    range = { 0 1 }    }
+	}
+	face_detail_eye_lower_lid_def = {
+        1 = { name = eye_lower_lid_def    range = { 0 1 }    }
+	}
+	face_detail_eye_socket = {
+        1 = { name = eye_socket_01    range = { 0 1 }    }
+		1 = { name = eye_socket_02    range = { 0 1 }    }
+		1 = { name = eye_socket_03    range = { 0 1 }    }
+	}
+	
+	#Body
+	gene_bs_body_type = {
+        25 = { name = body_fat_head_fat_low    range = { 0.3 0.4	  }    }
+		25 = { name = body_average    range = { 0.25 0.4	  }    }
+		25 = { name = body_fat_head_fat_medium    range = { 0.25 0.4	  }    }
+		25 = { name = body_fat_head_fat_full   range = { 0.25 0.4	  }    }
+		1 = { name = body_fat_head_fat_full   range = { 0.85 1	  }    } # "Gallywix Gene"
+    }
+	
+	### No tusks for goblins
+	gene_bs_mouth_tusks = {
+        1 = { name = mouth_tusks    range = { 0 0	  }    }
+    }
+	static_tusks = {
+        1 = { name = no_static_tusks    range = { 0 0	  }    }
+    }
+	
+	### No troll fingers for goblins
+	gene_bs_fingers = {
+        1 = { name = fingers_troll    range = { 0 0	  }    }
+    }
+	
+	###Other	
+	hairstyles = {
+		1 = { name = goblin_hairstyles 		range = { 0.0 1.0 } }
+	}
+	beards = {
+		1 = { name = no_beard 		range = { 0.0 1.0 } }
+	}
+	special_eyebrows = {
+        1 = {  name = no_eyebrows             range = { 0 1 }     }
+	}
+	face_detail_temple_def = {
+        1 = {  name = temple_def             range = { 0.2 0.8 }     }
+	}
+	expression_brow_wrinkles = {
+        1 = {  name = brow_wrinkles_01             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_02             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_03             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_04            range = { 0.5 1 }     }
+	}
+	expression_eye_wrinkles = {
+        1 = {  name = eye_wrinkles_01             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_02             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_03             range = { 0 1 }     }
+	}
+	expression_forehead_wrinkles = {
+        1 = {  name = forehead_wrinkles_01             range = { 0.1 1 }     }
+		1 = {  name = forehead_wrinkles_02             range = { 0.1 1 }     }
+		1 = {  name = forehead_wrinkles_03             range = { 0.1 1 }     }
+	}
+	complexion = {
+        1 = {  name = complexion_1             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_3             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_4             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_5             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_6             range = { 0.43 0.43 }     }
+		1 = {  name = complexion_7             range = { 0.43 0.43 }     }
+	}
```

Comment:
Assuming goblins didn't invent 💄.

Reply:
Oops. Fixed in 8939248fc1af428a97f830da0133c7a57ea09641.